src/ProjectManager.ts

```diff
@@ -66,7 +66,7 @@ class ProjectManager {
 
     this.workspaceTestState = {
       id: "root",
-      label: this.repoParser.type,
+      label: `${this.workspace.name} (${this.repoParser.type})`,
```

Comment:
Cool, I understand what is happening here.

What do you think about dropping the ` (${this.repoParser.type})`?  I never intended for that value to be displayed, it was more for debugging purposes.  I don't think it is offering anything meaningful. If people wanted to know what RepoParser had been used, we could log it out.

Reply:
I don't care about that, I left it there because I thought you (or the original author) cared for it